### DISCUSSION TITLE
Slider component/input

### DISCUSSION THREAD

**Discussion by kallefrombosnia** [area17/twill#1492] [Ideas]
As the title says, should we think about slider input in forms?

Since all of the components are made in Vue, I have searched for common popular and modular packages in Vue and it seems that [this](https://github.com/NightCatSama/vue-slider-component) package could be a good choice.

I would love to see this implementation since in past this could be a good choice for me, but I went with other 'alternatives' where sliders would perfectly fit.

**Comment by pboivin** [comment]
Good idea! I wonder if a small wrapper around [`<input type="range">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) would be a good solution too.

**Reply by kallefrombosnia** [reply]
Yeah, it would be good to stick to a default one, but I would rather ignore it. Reasons for that are feature lack and imprecise control of range point.

Feature lack is that it only supports one range point. If a user has a specific inner range, this would be insufficient.
Also without some external visual preview in form (with JS), it would be totally imprecise to select a specific required range.

So yeah, it sucks. Lemme know if I'm wrong in phrasing here. 🙂
